generator/containers.go

```diff
@@ -17,8 +17,18 @@ func (g *CaddyfileGenerator) getContainerCaddyfile(container *types.Container, l
 func (g *CaddyfileGenerator) getContainerIPAddresses(container *types.Container, logger *zap.Logger, ingress bool) ([]string, error) {
 	ips := []string{}
 
-	for _, network := range container.NetworkSettings.Networks {
-		if !ingress || g.ingressNetworks[network.NetworkID] {
+	ingressNetworkFromLabel, overrideNetwork := container.Labels[IngressNetworkLabel]
+
+	for networkName, network := range container.NetworkSettings.Networks {
+		include := false
```

Comment:
There might be a simpler way to express this logic, but I believe the `caddy_ingress_network` label needs to be a complete override of the allowed networks. Otherwise, we could still return IPs from another network?

Reply:
Will go with `onlyIngressIps`